e2e-tests/src/page-objects/crossmodel-explorer-view.ts

```diff
@@ -0,0 +1,11 @@
+/********************************************************************************
+ * Copyright (c) 2023 CrossBreeze.
+ ********************************************************************************/
+import { TheiaExplorerView } from '@theia/playwright';
+import { TheiaTabBarToolbar } from './theia-tabbar-toolbar';
+
+export class CrossModelExplorerView extends TheiaExplorerView {
+    async tabBarToolbar(): Promise<TheiaTabBarToolbar> {
```

Comment:
It seems to me like this call wouldn't need to be async. I'd also suggest to simply create it once and store it as a property of the view to be a bit more in line with how it is done in Theia but this is really minor as the tabbar-toolbar holds no state anyway.

Reply:
Good point, I refacted tabBarToolbar to be a public property, which is initialized in the constructor.

Is this what you also suggest?

Reply:
Yes, as a readonly public property should be fine, thank you!